### ISSUE TITLE
[BUG] checkip-server with SSL on non-standard port does not work

### ISSUE THREAD

**Issue by b-m-f** [troglobit/inadyn#419]
Scenario:
```
checkip-server: domain.com:8080
checkip-ssl: true
```

All requests will be sent to port 443, as it is the default HTTPS port.

What I would expect is that all traffic is sent via HTTPS, but to the non-standard port 8080, as defined in `checkip-server`.

**Comment by troglobit** [comment]
Which version of inadyn are you using? How did you build it, that is, does it have TLS support?

**Comment by b-m-f** [comment]
Its version `2.8.1`.

I am using it on TrueNas Scale, TLS works on standard HTTPS ports.

**Comment by troglobit** [comment]
Thought it rang familiar.  That release is over two years old, have you tried the latest release?

**Comment by b-m-f** [comment]
Hi @troglobit,

thanks for the quick feedback. Unfortunately the TrueNAS system does not allow for easy upgrades of individual binaries. I will open a bug request on their project for and would come back to you should the problem actually be with `inadyn` :+1:

**Comment by troglobit** [comment]
Yeah they need to upgrade their package of inadyn. Pretty sure this one is already fixed already. 

I'm closing this issue, but you're free to reopen it should the problem persist after they've updated.
